ocaml/xcp-rrdd/bin/rrdd/rrdd_http_handler.ml

```diff
@@ -5,6 +5,21 @@ open Rrdd_shared
 
 let with_lock = Xapi_stdext_threads.Threadext.Mutex.execute
 
+let rrd_handler (req : Http.Request.t) (s : Unix.file_descr) = function
+  | None ->
+      Http_svr.headers s (Http.http_404_missing ())
+  | Some rrd ->
+      let json = List.mem_assoc "json" req.Http.Request.query in
+      let headers =
+        List.concat
+          [
+            Http.http_200_ok ~version:"1.0" ~keep_alive:false ()
+          ; (if json then [] else [Http.Hdr.content_type ^ ": text/xml"])
```

Comment:
Should the content type not always be set?

Reply:
I'll add it, in a different commit since it's not being done in any endpoint, unlike text/xml